frontend/src/components/module-viewer/SideBarContentKiosk.tsx

```diff
@@ -40,20 +40,24 @@ const SideBarContentKiosk = (): React.ReactElement => {
           {ContentTypeCategories.map((type, i) => (
             <Container
               key={i}
-              onClick={() =>
-                setHideContent((prevState) => {
-                  const newState = {
-                    ...prevState,
-                  };
-                  newState[type] = !prevState[type];
-                  return newState;
-                })
-              }
               borderTop="1px"
               borderColor="background.grey"
               padding="0.5rem"
             >
-              <Flex>
+              <Flex
+                _hover={{
```

Comment:
Nit: you probably don't need to set this on hover, since cursors only apply on hover anyway.